src/commands/email/send.js

```diff
@@ -0,0 +1,136 @@
+const { flags } = require('@oclif/command');
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const sgMail = require('@sendgrid/mail');
+
+class Send extends BaseCommand {
+  async run() {
+    await super.run();
+    if (!process.env.SENDGRID_API_KEY) {
+      this.logger.error('Make sure you have an environment variable called SENDGRID_API_KEY set up with your SendGrid API key. Visit https://app.sendgrid.com/settings/api_keys to get an API key.');
+      return this.exit(1);
+    }
+    await this.promptForFromEmail();
+    const validFromEmail = this.validateEmail(this.fromEmail);
+    const stringFromEmail = validFromEmail.toString();
+    await this.promptForToEmail();
+    const validToEmail = this.validateEmail(this.toEmail);
+    await this.promptForSubject();
+    await this.promptForText();
+    const sendInformation = { to: validToEmail, from: stringFromEmail, subject: this.subjectLine, text: this.emailText, html: '<p>' + this.emailText + '</p>' };
+    await this.sendEmail(sendInformation);
+  }
+
+  validateEmail(email) {
+    var emailList = [];
+    var validEmail;
+    const multipleEmail = email.includes(',');
+    if (multipleEmail === true) {
+      emailList = email.split(',').map(item => {
+        return item.trim();
+      });
+    } else {
+      emailList[0] = email;
+    }
+
+    emailList.forEach(emailAddress => {
+      var re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
This logic should be stuck in a utility file since it's used in both commands (and is pretty complex).